src/processor/map/map_set.cpp

```diff
@@ -85,7 +86,11 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapSetNodeProperty(LogicalOperator
     for (auto& info : set->getInfos()) {
         executors.push_back(getNodeSetExecutor(info, *inSchema));
     }
-    auto printInfo = std::make_unique<OPPrintInfo>(set->getExpressionsForPrinting());
+    std::vector<binder::expression_pair> expressions;
+    for (auto i = 0u; i < set->getInfos().size(); ++i) {
```

Comment:
If we use a for loop like 
```
for (auto& info : set->getInfos) {
    expressions.push_back(info.setItem);
}
```
then we can avoid creating the getInfo(index) function